data/hlint.yaml

```diff
@@ -113,10 +113,9 @@
     - warn: {lhs: last (sortBy f x), rhs: maximumBy f x, side: isCompare f}
     - warn: {lhs: reverse (sortBy f x), rhs: sortBy (flip f) x, name: Avoid reverse, side: isCompare f, note: Stabilizes sort order}
     - warn: {lhs: sortBy (flip (comparing f)), rhs: sortOn (Down . f)}
```

Comment:
Sounds like this one also needs fixing/deleting?

Reply:
Covered in 86363703feaa7e1abf2d41fe458512f3943beab4.